data/default_imsim_configs

```diff
@@ -13,3 +13,6 @@ catalog = default
 
 [checkpointing]
 nobj = 1000
+
+[wl_params]
+gamma2_sign = -1
```

Comment:
So for this and the related gamma changes... I think this is right but probably it is good for @rmjarvis to look at it.

My understanding: the instance catalogs are now writing out "the phoSim convention". The entry is labeled_such (but nothing has changed from what it was doing before).  So if that is the case, when we read it in, the - sign is needed because the shear is applied by GalSim. The GalSim algorithm expects the "normal/DES" convention while the PhoSim shear algorithm doesn't.  So, this should give us the same physical orientation on the sensor. Correct?